libs/agora/services/src/lib/synapse-api.service.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-template-curly-in-string */
```

Comment:
Disabling this linting rule since we are ingesting wiki content from Synapse with this format.